frontend/src/metabase/common/components/FilterPicker/FilterOperatorPicker.tsx

```diff
@@ -0,0 +1,59 @@
+import type { ChangeEvent } from "react";
+import { useMemo } from "react";
+import Select from "metabase/core/components/Select";
+import * as Lib from "metabase-lib";
+
+type Option = {
+  name: string;
+  value: Lib.FilterOperatorName;
+};
+
+type FilterOperatorPickerProps = {
+  query: Lib.Query;
+  stageIndex: number;
+  column: Lib.ColumnMetadata;
+  value: Lib.FilterOperatorName | null;
+  onChange: (operator: Lib.FilterOperatorName) => void;
+};
+
+export function FilterOperatorPicker({
+  query,
+  stageIndex,
+  column,
+  value,
+  onChange,
+}: FilterOperatorPickerProps) {
+  const options: Option[] = useMemo(
+    () => getOptions(query, stageIndex, column),
+    [query, stageIndex, column],
+  );
+
+  return (
+    <div>
+      <Select
+        options={options}
+        value={value}
+        onChange={(e: ChangeEvent<HTMLSelectElement>) =>
+          onChange(e.target.value as Lib.FilterOperatorName)
+        }
+      />
```

Comment:
Couldn't use mantine select due to a CSP error.

Our internal select works with non-string values, but mantine select only works with string or numeric values - we would have to create some sort of external map to use opaque operator objects instead of operator names as our values here.